test/entry.ts

```diff
@@ -39,6 +39,22 @@ const structToObj = (struct: {}) => {
   return (fakeStructToObj || common.structToObj)(struct);
 };
 
+function withinExpectedTimeBoundaries(result?: Date): boolean {
```

Comment:
document what is considered expected time boundaries

Reply:
will do